ModuleUpdate.py

```diff
@@ -62,6 +62,9 @@ def update(yes=False, force=False):
     parser = argparse.ArgumentParser(description='Install archipelago requirements')
     parser.add_argument('-y', '--yes', dest='yes', action='store_true', help='answer "yes" to all questions')
     parser.add_argument('-f', '--force', dest='force', action='store_true', help='force update')
+    parser.add_argument('-add', '--additional_requirements_files', nargs="*", dest='additional_requirements',
+                        help='List paths to additional requirement files.')
```

Comment:
single dash with multiple letters is very uncommon. does not really matter, but i would suggest `'-a', '--append'` if this is open for discussion